### PR TITLE
Create registerErc20token function - offchain

### PR DESCRIPTION
**Pull request by jakcinmarina**

### CODE REVIEW

File: packages/tatum-core/src/model/request/RegisterErc20Token.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  public accountingCurrency?: Fiat
+
+  @Length(1, 150)
+  @IsNotEmpty()
+  public xpub: string
+
+  @IsNumber()
+  @IsNotEmpty()
+  @Min(1)
+  @Max(2147483647)
```
Comment (vlnevyhosteny) [inline packages/tatum-core/src/model/request/RegisterErc20Token.ts:66]: What is this magic number? https://en.wikipedia.org/wiki/2,147,483,647 ? 🙂 Can we make constant somewhere?

**Review by vlnevyhosteny — CHANGES_REQUESTED** [review]

File: packages/tatum-core/src/model/request/RegisterErc20Token.ts (hunk 1 above)
Comment (jakcinmarina) [reply packages/tatum-core/src/model/request/RegisterErc20Token.ts:66]: image

I got it from the docs, it's the max value for int32
we can but maybe in the other PR, what do you say?:)

Comment (vlnevyhosteny) [reply packages/tatum-core/src/model/request/RegisterErc20Token.ts:66]: Ok, I would just put somewhere `const MAX_INT32_VALUE = 2147483647;` and use it. Ok to do it later. 🙂

**Review by vlnevyhosteny — APPROVED** [review]

File: packages/tatum-core/src/model/request/RegisterErc20Token.ts (hunk 1 above)
Comment (samuelsramko) [reply packages/tatum-core/src/model/request/RegisterErc20Token.ts:66]: 2^32-1

**Pull request closed without merging**